packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -580,7 +622,9 @@ export default decorate([
             get(() => hostsById[poolsById[$container].master].version)
         ),
       srPredicate: ({ srs }) => sr => isSrWritable(sr) && !includes(srs, sr.id),
-      remotePredicate: ({ remotes }) => ({ id }) => !includes(remotes, id),
+      remotePredicate: ({ proxyId, remotes }) => remote =>
+        remotes.indexOf(remote.id) === -1 &&
```

Comment:
`remotes.includes(remote.id)`  ?

Reply:
`indexOf` perform well than `includes`.

Reply:
`includes` checks if a regex is passed in opposite of `indexOf` which accept it.

Reply:
So the performance difference is very negligible and `includes` says more about what you actually want.